apps/accounts/signal_handlers.py

```diff
@@ -91,3 +94,18 @@ def delete_to_vault(sender, instance, **kwargs):
 for model in (Account, AccountTemplate, Account.history.model):
     post_save.connect(VaultSignalHandler.save_to_vault, sender=model)
     post_delete.connect(VaultSignalHandler.delete_to_vault, sender=model)
+
+
+class VaultPubSub(LazyObject):
+    def _setup(self):
+        self._wrapped = RedisPubSub('refresh_vault')
+
+
+vault_pub_sub = VaultPubSub()
+
+
+@receiver(django_ready)
+def subscribe_vault_change(sender, **kwargs):
+    logger.debug("Start subscribe vault change")
+
+    vault_pub_sub.subscribe(lambda name: refresh_vault_client())
```

Comment:
No major issues found, but here's some suggestions:

  * The 'django_ready' setting is not used at all in this file which should be moved outside the 'audit_signal_handler'. 
  * Move the creation of DjangoReadyHandler to signal handler. This will save us time when implementing another signal.